Docs/source/running_cpp/parameters.rst

```diff
@@ -1012,6 +1012,32 @@ Diagnostics and output
     Use a negative number to disable data dumping.
     This is ``-1`` (disabled) by default.
 
+    * ``<species_name>.random_fraction`` (`float`) optional
```

Comment:
alignment seams a bit off.

Shouldn't those options be sub-options of plot and reduced diagnostics functions?

Reply:
Currently, this particle filter only works for plotfiles. So I intended to put them as a sub-options after `amr.plot_int`. Shouldn't these codes give what I want?

Reply:
@ax3l @MaxThevenet What alignments do you guys want?

Reply:
What we mean is: there are too many spaces before the `*`.

You want to create this in the docs:
* ``amr.plot_int`` (`integer`) optional
* ``<species_name>.random_fraction`` (`float`) optional 

And not:

* ``amr.plot_int`` (`integer`) optional
     * ``<species_name>.random_fraction`` (`float`) optional